packages/elements/test/unit/components/breadcrumb.component.spec.ts

```diff
@@ -59,6 +59,14 @@ describe(MutationTestReportBreadcrumbComponent.name, () => {
     expect(elements[2].querySelector('a')).null;
   });
 
+  it('should dispatch open-file-picker event', () => {
+    // Act
+    const event = sut.catchCustomEvent('mte-file-picker-open', () => void sut.element.shadowRoot?.querySelector('button')?.click());
```

Comment:
Doesn't the `void` here discard the promise? You still want the `act` function to be awaited to not run into any race conditions